index.html

```diff
@@ -10,129 +10,129 @@
-      <nav class="work">
-        <a href="#"><img src="images/Tags.jpg"></a>
-      </nav>
-      <nav>
-        <button type="button" class="button">See Project</button>
-      </nav>
-    </div>
-
-    <div class="page-5">
-      <div class="container">
-        <a href="#"><img src="images/Snapshoot Portfolio 4.jpg"></a>
-      </div>
-      <div class="heading-2">
+        <ul class="column">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="works">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul> 
+        <nav>
+          <button type="button" class="button">See Project</button>
+        </nav>
+      </section>
+  
+      <section class="page-5">
+        <img class="snapshoot" id="snapshoot-R" src="images/Snapshoot Portfolio 4.jpg">
         <h2>Multi-Post <br> Stories</h2>
-      </div>
-      <ul class="column">CANOPY
-        <li><a href="#"><img src="images/point.jpg"></a></li>
-        <li><a href="#"><img src="images/back.jpg"></a></li>
-        <li><a href="#"><img src="images/point.jpg"></a></li>
-        <li><a href="#"><img src="images/year.jpg"></a></li>
-      </ul>
-      <p>A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
-      <nav class="work">
-        <a href="#"><img src="images/Tags.jpg"></a>
-      </nav>
-      <nav>
-        <button type="button" class="button">See Project</button>
-      </nav>
-    </div>
-
-    <div class="about-myself">
-      <div class="heading-3">
-        <h2>About <br> Myself</h2>
-      </div> 
-      <p>Hello I'm a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me.</p>
-      <a class="links">LET'S CONNECT</a>
+        <ul class="column">CANOPY
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/back.jpg"></a></li>
+          <li><a href="#"><img src="images/point.jpg"></a></li>
+          <li><a href="#"><img src="images/year.jpg"></a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="works">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul> 
+        <nav>
+          <button type="button" class="button">See Project</button>
+        </nav>
+      </section>
+    </section>
+  
+    <section class="about-myself">
```

Comment:
- [ ] [OPTIONAL] Good job adding the about me section 💯, but there is still an issue, it could be better if you fix the right margin, to make it similar to Figma design. 🎲 

![image](https://user-images.githubusercontent.com/91301423/201475271-e8267907-c9f4-440d-9da2-a036a6a18678.png)